src/Testing/InteractsWithEmails.php

```diff
@@ -93,6 +93,19 @@ public function seeEmail($email, $queued = MailGrasp::UNQUEUED)
 
         return $this;
     }
+    
+    public function dontSeeEmail($email, $queued = MailGrasp::UNQUEUED)
+    {
+        if ($queued) {
+            $found = $this->mailer->getQueuedEmail($email) ? true : false;
+        } else {
+            $found = $this->mailer->getEmail($email) ? true : false;
+        }
+
+        $this->assertFalse($found, $this->mailer->getError($email));
```

Comment:
The second parameter is the error message returned by phpunit. 
mailer->getError() will print "expected $email founded $list-of-emails-sent" 
Could you write a different error message? Thanks.